labs/lab18-helm-charts/README.md

```diff
@@ -0,0 +1,435 @@
+
+```bash
+cd ../../..
+```
+If you haven't already, set your environment variables:
+
+```bash
+export REPLICATED_APP=...
+export REPLICATED_API_TOKEN=...
+```
+
+We need a directory that will contain the Helm Chart and the Replicated manifests. Let's create a directory for our application, and in there a `manifests` directory that will contain the manifests and helm chart.
+
+```bash
+mkdir helm-wordpress
+cd helm-wordpress
+mkdir manifests
+```
+
+We need to move the helm chart package we created previously to the manifests directory we just created. Depending on where your helm chart was created, the path in the example command below may be different.
+
+```bash
+mv ../charts/bitnami/wordpress/wordpress-14.0.7.tgz manifests
+```
+
+Next, let's create a couple of manifests to finish our first release. Both of these manifests are Replicated [Custom Resources](https://docs.replicated.com/reference/custom-resource-about) and should be created in the `manifests` directory.
+
+#### Adding kots-app.yaml
+
+The first Custom Resource we'll create is [Application](https://docs.replicated.com/reference/custom-resource-application). This manifest includes details about the application we created.
+
+``` yaml
+#kots-app.yaml
+apiVersion: kots.io/v1beta1
+kind: Application
+metadata:
+  name: helm-wordpress
+spec:
+  title: Helm Wordpress
+  icon: https://upload.wikimedia.org/wikipedia/commons/9/93/Wordpress_Blue_logo.png
+  statusInformers:
+    - deployment/wordpress
+
+```
+
+#### Adding wordpress.yaml
+
+The second Custom Resource we'll create is [Helm Chart](https://docs.replicated.com/reference/custom-resource-helmchart). This file has information about the Wordpress Helm Chart. This file is specific to the Helm Chart, so if your application contains multiple, separate charts, you will need a manifest for each Helm Chart. You do not need to define a `Helm Chart` manifest for any chart included as a sub-chart.
+
+```yaml
+#wordpress.yaml
+apiVersion: kots.io/v1beta1
+kind: HelmChart
+metadata:
+  name: wordpress
+spec:
+  # chart identifies a matching chart from a .tgz
+  chart:
+    name: wordpress
+    chartVersion: "14.0.7"
+  # helmVersion identifies the Helm Version used to render the chart. Default is v2.
+  helmVersion: v3
+
+  # useHelmInstall identifies whether this Helm chart will use the
+  # Replicated Helm installation (false) or native Helm installation (true). Default is false.
+  # Native Helm installations are only available for Helm v3 charts.
+  useHelmInstall: true
+
+  # weight determines the order that charts with "useHelmInstall: true" are applied, with lower weights first.
+  weight: 0
+
+  # values are used in the customer environment, as a pre-render step
+  # these values will be supplied to helm template
+  values: {}
+```
+
+Before we create our release, let's initialize git. We will use a command line option that uses Git metadata to determine the version label, channel to promote to and more.
```

Comment:
```suggestion
Before we create our release, let's initialize git. We will use a command line option that uses Git metadata to determine the version label, channel to promote to and more.
From the `repl-helm-lab/helm-wordpress` directory, run the following.
```

Just wanna be extra clear here since doing this wrong becomes a pain to back out